ATLauncher/ATLauncher
# [Architecture] HierarchyPanel : NewsTab

### Description of the Change

Implement the foundations for memory management via life cycle in accordance to the view being seen or not.

~~This PR is not eligible for merge until all view models are created and handled. This should be done after one step is complete.~~
This PR is needed for PacksTab and to remove TabChangeManager

### Testing

- Open launcher, Go to news tab, then leave news tab, and go back to news tab.
- Run check for updates

### Related Issues

#623
